dom-creation/main.js

```diff
@@ -59,26 +59,35 @@ var pokedex = [
 
 function renderPokemon(pokemon) {
   var $columnThird = document.querySelector('.row');
+
   var div = document.createElement('div');
   div.setAttribute('class', 'column-third');
+
   var card = div.appendChild(document.createElement('div'));
   card.setAttribute('class', 'pokemon-card');
+
   var imageUrl = card.appendChild(document.createElement('img'));
   imageUrl.setAttribute('src', pokemon.imageUrl);
+
   var pokemonText = card.appendChild(document.createElement('div'));
   pokemonText.setAttribute('class', 'pokemon-card-text');
+
   var name = pokemonText.appendChild(document.createElement('h2'));
   name.textContent = pokemon.name;
+
   var number = pokemonText.appendChild(document.createElement('h3'));
   number.textContent = pokemon.number;
+
   var description = pokemonText.appendChild(document.createElement('p'));
   description.textContent = pokemon.description;
-  $columnThird.appendChild(div);
+
+  var output = $columnThird.appendChild(div);
+  return output;
```

Comment:
Not quite, you should delete the `$columnThird` variable, the `div` variable is the one you want to return here.